### DISCUSSION TITLE
Odd results with Manual Mesh Bed leveling

### DISCUSSION THREAD

**Discussion by wbvczar** [Jyers/Marlin#196] [Q&A] [answered]
I have been using the "ManualMesh_3x3" pre-compiled firmware, but I wanted to compile my own for practice.  I used the ManualMesh-3x3 files from the "Marlin-Ender_3_V2_Extensible_UI/Configuration Files/E3V2 Templates/" folder.  Everything compiled fine and I flashed the firmware to my printer.

The oddity started when I tried to level the bed with the "Level" section of the menu.  I had already leveled the bed on the 4 corners with a piece of paper, so they should have been very close.  But when It went to each point of the 3x3 grid, I would drop the Z offset to 0 (from the 0.2 default value) and it would still be way off.  On a few points I had to drop it to -.5 and on others it would go to -1 and still not be touching the paper.  It was like it wasn't lower on the Z access.

So, I went back to the pre-compiled version, and the Level menu worked as it had before and I was able to properly manual mesh level the bed.

I did do a fresh download of the code today before compiling, so I wasn't sure if something had changed in the code that would make it different than the 1.1.1 pre-compiled code.

**Comment by Jyers** [comment, marked as answer]
I just took a look and immediately found the bug. Looks like one of the recent PR contributors made a mistake with a conditional and disabled some of the manual mesh code. It should be fixed with the latest commit, sorry for the confusion.

**Reply by wbvczar** [reply]
Thank you for looking into this, and thank you for your work.  It is really appreciated!
